projects/1.20/assets/tetra/tetra/lang/zh_cn.json

```
@@ -0,0 +1,2465 @@
+  "tetra.holo.craft.materials.stat.tool_level": "工具等级",
+  "tetra.holo.craft.materials.stat.tool_level.short": "等级",
+  "tetra.holo.craft.materials.stat.tool_level.tooltip": "使用这种材料制作的组件的基础等级",
+  "tetra.holo.craft.materials.stat.tool_efficiency": "工具效率",
+  "tetra.holo.craft.materials.stat.tool_efficiency.short": "效率",
+  "tetra.holo.craft.materials.stat.tool_efficiency.tooltip": "使用这种材料制作的组件的基础工具效率",
+  "tetra.holo.craft.materials.stat.durability": "耐久度",
+  "tetra.holo.craft.materials.stat.durability.short": "耐久",
+  "tetra.holo.craft.materials.stat.durability.tooltip": "使用这种材料制作的组件的基础耐久度",
+  "tetra.holo.craft.materials.stat.integrity": "完整度",
+  "tetra.holo.craft.materials.stat.integrity.short": "完整度",
+  "tetra.holo.craft.materials.stat.integrity.tooltip": "使用这种材料制作的组件的基础完整度增益或消耗（增益丨消耗）",
+  "tetra.holo.craft.materials.stat.magic_capacity": "魔法容量",
+  "tetra.holo.craft.materials.stat.magic_capacity.short": "魔力",
+  "tetra.holo.craft.materials.stat.magic_capacity.tooltip": "使用这种材料制作的主要组件的魔法容量",
+  "tetra.holo.craft.materials.stat_modifier": "强化",
+  "tetra.holo.craft.materials.stat_modifier.tooltip": "使用这种材料制作的主要组件会提供§e%s§r强化：",
+  "tetra.holo.craft.materials.stat_effect": "效果",
+  "tetra.holo.craft.materials.stat_effect.tooltip": "使用这种材料制作的组件会提供§e%s§r效果：",
+  "tetra.holo.craft.materials.feature": "特性",
+  "tetra.holo.scan.status": "§8状态：§r%s",
+  "tetra.holo.scan.active": "激活",
+  "tetra.holo.scan.snoozed": "§8休眠 §r%s:%s",
+  "tetra.holo.quick_access.snooze_on": "§7使扫描器\n§r休眠",
+  "tetra.holo.quick_access.snooze_off": "§7使扫描器\n§r启动",
+  "tetra.holo.place": "§7[§8Shift+右键§7] §8放置",
+  "tetra.workbench.slot_detail.details_tab": "详情",
+  "tetra.workbench.slot_detail.craft_tab": "制作",
+  "tetra.workbench.slot_detail.tweak_tab": "调整",
+  "tetra.workbench.slot_detail.tweak_apply": "应用",
+  "tetra.workbench.module_detail.empty": "空槽位",
+  "tetra.workbench.module_detail.not_tweakable": "调整不可用",
+  "tetra.workbench.schematic_list.empty": "所选插槽没有可用的升级",
+  "tetra.workbench.schematic_detail.craft": "制作",
+  "tetra.workbench.schematic_detail.back": "返回",
+  "tetra.workbench.schematic_detail.experience": "此操作需要%d级经验",
+  "tetra.workbench.schematic_detail.destabilize_tooltip": "该物品有魔力去稳化的组件，进行此操作有可能对以下组件造成负面效果：",
+  "tetra.workbench.schematic_detail.repair_tooltip": "替换修复循环中的下一个组件，该物品会被§e完全修复",
+  "tetra.workbench.schematic_detail.no_material_tooltip": "该组件需要材料来制作",
+  "tetra.workbench.schematic_detail.material_count_tooltip": "提供的材料不足",
+  "tetra.workbench.schematic_detail.material_tooltip": "提供的材料不能用于制作该组件",
+  "tetra.workbench.schematic_detail.integrity_tooltip": "制作这个会超出物品的完整度上限",
+  "tetra.workbench.schematic_detail.tools_tooltip": "没有此操作所需的工具",
+  "tetra.workbench.schematic_detail.level_tooltip": "制作这个需要更多经验",
+  "tetra.workbench.schematic_list.temporary": "可以制作，因为你的背包中有可用物品，或是附近有工艺图纸",
+  "tetra.sources.multi_tooltip": "此工艺由§9%s§r添加，再由§9%s§r修改或覆写",
+  "tetra.sources.single_tooltip": "此工艺由§9%s§r添加",

```

| **审查级别**    |
|:-:|
| ❓疑问 / 需要作者确认 |

**原文: This schematic was added by §9%s**

```suggestion
  "tetra.sources.single_tooltip": "此工艺由§9%s§r添加",
```